eng/pipelines/libraries/base-job.yml

```diff
@@ -65,19 +69,26 @@ jobs:
         - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
           - _msbuildCommonParameters: /p:OfficialBuildId=$(Build.BuildNumber)
 
+        - _coreClrArtifactName: ''
+        - _coreClrDownloadPath: ''
+        - _coreClrOverridePathArg: ''
+
+        - ${{ if eq(parameters.useLiveCoreClr, true) }}:
+          - _coreClrDownloadPath: '$(Build.SourcesDirectory)/artifacts/transport/coreclr'
+          - _coreClrArtifactName: 'CoreCLRProduct_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.buildConfig }}'
```

Comment:
Right now if I want to build a debug libraries build I would need to build coreclr for debug. Maybe we should accept a parameter which is `coreclrBuildConfig` in order to download the right artifact and depend on the right build.